lms/wsgi.py

```diff
@@ -1,3 +1,7 @@
+"""
+WSGI file for LMS
```

Comment:
It might be nice to have a little explanation here of what that means.  This pretty much just repeats the file name.